tr1d1um.yaml

```
@@ -123,6 +123,53 @@ log:
 # information using Argus.
 # Optional: if key is not supplied, webhooks would be disabled.
 webhook:
+
+  # validation provides options for validating the webhook's URL and TTL related fields.
+  validation:
+
+    # url will check the webhook's Config.URL, FailureURL, and Config.AlternativeURLs.
+    url:
+      # httpsOnly will allow only URLs with https schemes through if true.
+      # (Optional). Defaults to 'false'.
+      httpsOnly: false
+
+      # allowLoopback will disallow any canonical or IP loopback address if false.
+      # (Optional). Defaults to 'false'.
+      allowLoopback: true
+
+      # allowIP will disallow any IP addresses through if false.
+      # (Optional). Defaults to 'false'.
+      allowIP: true
+
+      # allowSpecialUseHosts, if false, will disallow URLs that contain any string in invalidHosts or any string
+      # in SpecialUseHosts in webhookValidationConfig.go .
+      # (Optional). Defaults to 'false'.
+      allowSpecialUseHosts: true
+
+      # allowSpecialUseIPs, if false, will disallow URLs that contain any string in invalidSubnets
+      # or any string in SpecialUseIPs in webhookValidationConfig.go .
+      # (Optional). Defaults to 'false'.
+      allowSpecialUseIPs: true
+
+      # invalidHosts is a slice that contains strings of hosts that we do not want allowed in URLs
+      # if allowSpecialUseHosts is false.
+      # (Optional). Defaults to an empty slice.
+      invalidHosts: []
+
+      # invalidSubnets is a slice of strings of IPs that will be disallows in URLs if allowSpecialUseIPs 
+      # is false.
+      # (Optional). Defaults to an empty slice.
+      invalidSubnets: []
+
+    # ttl is the webhook's time to live.
+    ttl:
+      # max is the length of time a webhook is allowed to live.
+      max: 1m
+
+      # jitter is the amount of additional time given to the webhook's TTL to account for
+      # any variations in delivery.
```

```suggestion
      # jitter is the buffer time added when checking that the Until value is valid.  If there is a slight clock skew between servers or some delay in the http request, jitter should help account for that when ensuring that Until is not a time too far in the future.
```
The original comment makes it sound like we modify the webhook.Until value with the jitter before storing the webhook in argus.  My suggestion makes it more clear that jitter is only being used for validation purposes.